### ISSUE TITLE
IOS support

### ISSUE THREAD

**Issue by wpp1000** [MiniDude22/Pandora-Downloader#16]
Anyway there could be support added back to IOS 11, making the app 64bit. Or if I could have the original source code for the IOS app I would do it.

**Comment by DerrikMilligan** [comment]
This never ran on iOS11? It runs on Mac fine I'm sure.

**Comment by wpp1000** [comment]
It ran on previous versions of iOS. but they dropped 32 bit support, so it does not currently work on iOS 11 and up. Just wondering if you would update it to 64 bit.

**Comment by DerrikMilligan** [comment]
How were you running it on iOS? What did you use to run the flask server? What are the errors you're getting? I feel like you're making this up and just looking for someone to find a way to run it on iOS. A change from x86 to x64 wouldn't affect this.

**Comment by wpp1000** [comment]
There is a cydia tweak on iOS called Pandora Downloader. I just assumed this was the source code for the app and the same developer. But I guess not.

**Comment by hqker** [comment]
@wpp1000, You are correct. This is the source code for said Cydia tweak. Furthermore, it was compatible for iOS devices on 32 but infrastructure. However, as you have mentioned, it doesn’t currently run on 64 but infrastructure. I too, would like to see this updated if the developer would be willing to. I’m sure many would appreciate it. I might consider the project but there’s much to be learned about current structure and iOS 11 software updates in order for me to make it happen. Hopefully the developer will update it, as I’m sure they are more capable of doing so as it is their project after all.

**Comment by DerrikMilligan** [comment]
This is not the Cydia tweak and does not work on IOS.

**Comment by hqker** [comment]
So this project is not this?: http://topcydiasources.com/tag/pandora-downloader/
If it is, then it does work for downloading Pandora songs from Pandora on iOS devices running iOS 8 through iOS 8.1.3, as I have it running on my older iPhone with one of those firmwares.
I realize that your project is for a Chrome Extension, so I'm sure there are some who will assume that it is.
However, if this is not the same project as the above mentioned, then my apologies and I hope you enjoy your day. :)

**Comment by MiniDude22** [comment]
A chrome extension has nothing to do with running on IOS. Chrome doesn't even allow extensions on mobile devices whether iOS or Android. Plus this requires running a local webserver to download and tag the music. Sorry they used the same name as my project.

**Comment by hqker** [comment]
I understand that a chrome extension has nothing to do with running on iOS and that chrome doesn't support extensions on any mobile platform yet. I was just confirming that they were separate projects.
Your project, this project, is for a chrome extension that has the same name as another project that someone else (apparently) has made by the same name for iOS users who are Jailbroken to download Pandora songs. Hopefully, anyone else that sees this thread who came from the link I did that says that they are one in the same will stop at this post and understand that they do not have anything to do with each other. Someone has simply referenced incorrectly to your project. I can try to ask the reference from that source to remove his link to your project or edit his reference to the actual reference that it pertains to (if he wishes) if you wish to prevent others from coming here mistakenly for something else.

**Comment by MiniDude22** [comment]
Okay thanks. Sorry he linked to my project. I wouldn't trust that software without seeing it's source code.

**Comment by hqker** [comment]
My pleasure, and no worries. **I'll see** if I can have him remove his link so that others seeking a fix for something you can't help them with won't come here bugging you. I was actually trying to obtain the source code for it so that I could see it for myself, that's how I ended up here. Anywho, I hope you enjoy your day and glad we could get this resolved ^-^

**Comment by MiniDude22** [comment]
Thanks for your help. I was very confused why people were asking about iOS. Hope you have a good day too.
